### ISSUE TITLE
CVE-2020-36079 clarification needed

### ISSUE THREAD

**Issue by fgeek** [zenphoto/zenphoto#1292]
[CVE-2020-36079](https://nvd.nist.gov/vuln/detail/CVE-2020-36079) vulnerability description is:
```
Zenphoto through 1.5.7 is affected by authenticated arbitrary file upload, leading to remote code execution. The attacker must navigate to the uploader plugin, check the elFinder box, and then drag and drop files into the Files(elFinder) portion of the UI. This can, for example, place a .php file in the server's uploaded/ directory.
```
Am I correct that this is not a vulnerability and needs admin privileges (or special permissions for regular users) and that the software is intended to behave in this manner?

**Comment by sbillard** [comment]
Specifically, the user must be logged in and have upload rights. So, don't give those rights to someone you do not trust.

**Comment by acrylian** [comment]
We just learned about this as well. This actually had been reported to us last year. And we addressed this issue in 1.5.7 
https://www.zenphoto.org/news/zenphoto-1.5.7/ and just addressed this on our news section: https://www.zenphoto.org/news/why-not-every-security-issue-is-really-an-issue/

Actually you need full admin rights to upload executable files with the elFinder plugin now and not just upload rights.
